### ISSUE TITLE
Image utilities are unreachable as they are behind the add button

### ISSUE THREAD

**Issue by mayeenulislam** [UniSharp/laravel-filemanager#658]
When the window fills with images, directories, the last item on the right becomes unmanageable, because of the floating add button. We can't do anything with that particular item until removing any other, or adding a new to lengthen the window.

And provide the followings :
 * Operating system : Windows 7 Ultimate
 * Laravel version : 5.6.2
 * Package version : 1.8.0
 * Steps to reproduce your issue :

1. Open Laravel FileManager modal
2. Add images as much as they cover the entire window
3. Try to delete the last item on the right just underneath the add button, you can't

**Screenshot**<br>
![image-utilities-are-not-available](https://user-images.githubusercontent.com/4551598/43755422-a819cdba-9a31-11e8-81b1-b3023093f48f.png)

 * Screenshots of browser console :
![console](https://user-images.githubusercontent.com/4551598/43755411-9b118220-9a31-11e8-800a-fb09aded72ea.png)

## Possible Workaround
As a designer I can propose the following position just beside the header:<br>
![proposed-layout](https://user-images.githubusercontent.com/4551598/43755679-e0ec6de0-9a32-11e8-882b-32c28b0fb5a7.png)

With two simple `<button>` with and some bootstrap class will do the trick:

```html
<div class="panel-heading">
      <button class="btn btn-md btn-default pull-right"><i class="fa fa-image" aria-hidden="true"></i> Add Images</button>
      <button class="btn btn-md btn-default pull-right"><i class="fa fa-folder" aria-hidden="true"></i> Add Directory</button>
</div>
```
And if you want to make more ease, you can use the `position: static` or Affix plugin from Bootstrap 3 to the `.panel-heading`.

**Comment by g0110280** [comment]
This is really an annoying problem. Thanks for your advice @mayeenulislam ! We will fix this in `v2.0` version.

**Comment by g0110280** [comment]
Fixed in `v2.0.0-alpha6`.
